documentation/documentation.gradle.kts

```diff
@@ -18,7 +18,7 @@ modularProjects.forEach { evaluationDependsOn(it.path) }
 
 javaLibrary {
 	mainJavaVersion = JavaVersion.VERSION_1_8
-	testJavaVersion = JavaVersion.VERSION_1_8
+	testJavaVersion = JavaVersion.VERSION_11
```

Comment:
Because junit-platform-jfr targets Java 11 -- will be reverted when we turn it into a MR-JAR with base 8.

Reply:
Fixed via an extra configuration.